beeline-spring-boot-sleuth-starter/src/main/java/io/honeycomb/beeline/spring/beans/BraveBeelineReporter.java

```diff
@@ -60,11 +60,18 @@ private void adaptBraveAnnotationToHoneycombSpanEvent(final Span span, final io.
     }
 
     private io.honeycomb.beeline.tracing.Span transformBraveSpanToHoneycombSpan(final Span span) {
-        final PropagationContext propagationContext = new PropagationContext(span.traceId(), span.id(), null, null);
-        final io.honeycomb.beeline.tracing.Span hcRootSpan = beeline.startTrace(span.name(), propagationContext, properties.getServiceName());
-        final long startTimestamp = span.timestamp();
-        if (startTimestamp > 0) {
-            hcRootSpan.markStart(startTimestamp, startTimestamp);
+        final PropagationContext propagationContext = new PropagationContext(span.traceId(), span.parentId(), null, null);
+        final io.honeycomb.beeline.tracing.Span hcRootSpan = beeline.getTracer().startTrace(
+            beeline.getSpanBuilderFactory().createBuilder()
+                .setSpanName(span.name())
+                .setServiceName(properties.getServiceName())
+                .setParentContext(propagationContext)
+                .setSpanId(span.id())
+                .build()
+        );
+        final Long startTimestamp = span.timestamp();
+        if (startTimestamp != null && startTimestamp > 0) {
+            hcRootSpan.markStart(startTimestamp/1000, startTimestamp/1000);
```

Comment:
Should these constants be `MICROS_IN_MILLISECOND`?

Reply:
Yup, you're right. There is a const we can use from above. Thanks!